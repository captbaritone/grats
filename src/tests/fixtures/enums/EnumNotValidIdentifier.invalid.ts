/** @gqlType */
class SomeType {
  /** @gqlField */
  hello: string;
}

/** @gqlEnum */
export enum Enum {
  VALID = "VALID",
  INVALID = "NOT AT ALL VALID",
}
