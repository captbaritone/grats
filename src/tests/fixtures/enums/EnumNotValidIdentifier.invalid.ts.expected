-----------------
INPUT
----------------- 
/** @gqlType */
class SomeType {
  /** @gqlField */
  hello: string;
}

/** @gqlEnum */
export enum Enum {
  VALID = "VALID",
  INVALID = "NOT AT ALL VALID",
}

-----------------
OUTPUT
-----------------
src/tests/fixtures/enums/EnumNotValidIdentifier.invalid.ts:10:13 - error: Names must only contain [_a-zA-Z0-9] but "NOT AT ALL VALID" does not.

10   INVALID = "NOT AT ALL VALID",
               ~~~~~~~~~~~~~~~~~~
